pkg/apis/certmanager/v1/types.go

```diff
@@ -233,6 +233,21 @@ const (
 	UsageNetscapeSGC       KeyUsage = "netscape sgc"
 )
 
+// Keystore specific secret keys
+const (
+	// Pkcs12SecretKey is the name of the data entry in the Secret resource
+	// used to store the p12 file.
+	Pkcs12SecretKey = "keystore.p12"
+	// Data Entry Name in the Secret resource for PKCS12 containing Certificate Authority
+	Pkcs12TruststoreKey = "truststore.p12"
+
+	// JksSecretKey is the name of the data entry in the Secret resource
+	// used to store the jks file.
+	JksSecretKey = "keystore.jks"
+	// Data Entry Name in the Secret resource for JKS containing Certificate Authority
+	JksTruststoreKey = "truststore.jks"
+)
```

Comment:
One final nitpick is that our house style is that acronyms in these variable names should be all-caps

E.g. `JKSTruststoreKey`

Existing examples can be seen higher up in this file:

`IPSANAnnotationKey`
`URISANAnnotationKey`



https://github.com/cert-manager/cert-manager/pull/5597/files#diff-fa12e017e1273e09efc9b417e8b02442c7aba764e9c1b81cd0e93fb35700f9f1R236-R249